apps/authentication-service/cypress/support/commands.ts

```diff
@@ -39,29 +40,36 @@ import { JwtPayload } from "jsonwebtoken";
 //   }
 // }
 
-Cypress.Commands.add("loginClientAuth0", () => {
+Cypress.Commands.add("loginClientAuth0", (doUseRoleUser: boolean = true) => {
```

Comment:
I'd use should instead of do

Reply:
I got rid of this by refactoring the login code, as you suggested.